src/client/driver/index.js

```diff
@@ -173,6 +174,16 @@ export default class ClientDriver {
             });
     }
 
+    _onResizeWindowCommand (command) {
```

Comment:
Why we need to run it through client? Can't it be implemented just server-side, like `wait`?